cards.js

```diff
@@ -17,7 +17,100 @@ function shuffle(array){
 }
 
 function buildDeck(){
-
+  var deck = [];
+  for (var i=0; i<suits.length; i++) {
```

Comment:
There are a few places where coding style is inconsistent. I'll make notes where I see them, but they don't affect functionality or completeness of your code.

For example, we typically want to pad operators with a space, so this line becomes: ``` for (var i = 0; i < suits.length; i++) {```